source/sio/io_uring/file_handle.hpp

```diff
@@ -484,6 +486,87 @@ namespace sio::io_uring {
     }
   };
 
+  struct acceptor : native_fd_handle {
+    ip::endpoint local_endpoint_;
+
+    acceptor(exec::io_uring_context& context, int fd, const ip::endpoint& local_endpoint) noexcept
+      : native_fd_handle(context, fd)
+      , local_endpoint_(local_endpoint) {
+    }
+  };
+
+  struct accept_submission {
+    int fd_;
+    ip::endpoint local_endpoint_;
+
+    accept_submission(int fd, ip::endpoint local_endpoint) noexcept
+      : fd_{fd}
+      , local_endpoint_(static_cast<ip::endpoint&&>(local_endpoint)) {
+    }
+
+    static constexpr std::false_type ready() noexcept {
+      return {};
+    }
+
+    void submit(::io_uring_sqe& sqe) const noexcept;
+  };
+
+  template <class Receiver>
+  struct accept_operation_base
+    : stoppable_op_base<Receiver>
+    , accept_submission {
+    accept_operation_base(
+      exec::io_uring_context& context,
+      Receiver receiver,
+      int fd,
+      ip::endpoint local_endpoint) noexcept
+      : stoppable_op_base<Receiver>{context, static_cast<Receiver&&>(receiver)}
+      , accept_submission{fd, static_cast<ip::endpoint&&>(local_endpoint)} {
+    }
+
+    void complete(const ::io_uring_cqe& cqe) noexcept {
+      if (cqe.res >= 0) {
+        stdexec::set_value(
+          static_cast<accept_operation_base&&>(*this).receiver(),
+          byte_stream{
+            native_fd_handle{this->context(), cqe.res}
+        });
+      } else {
+        SIO_ASSERT(cqe.res < 0);
+        stdexec::set_error(
+          static_cast<accept_operation_base&&>(*this).receiver(),
+          std::error_code(-cqe.res, std::system_category()));
+      }
+    }
+  };
+
+  template <class Receiver>
+  using accept_operation = stoppable_task_facade<accept_operation_base<Receiver>>;
+
+  struct accept_sender {
+    using is_sender = void;
+
+    using completion_signatures = stdexec::completion_signatures<
+      stdexec::set_value_t(byte_stream&&),
```

Comment:
In completion sigs, I prefer to use rvalue to indicate that I moved some object from a storage. I think, here `stdexec::set_value_t(byte_stream)`, later `stdexec::set_value_t(socket_handle)` is enough